system/netdata-openrc.in

```diff
@@ -25,7 +25,7 @@ extra_started_commands="getconf"
 pidfile="/run/netdata/netdata.pid"
 command="${NETDATA_INSTALL_PATH}/usr/sbin/netdata"
 command_background="yes"
-command_args="${NETDATA_EXTRA_ARGS}"
+command_args="-pidfile ${pidfile} ${NETDATA_EXTRA_ARGS}"
```

Comment:
prefix should be:
```
extra_started_commands="getconf"
pidfile="/run/netdata.pid"
command="${NETDATA_INSTALL_PATH}/usr/sbin/netdata"
command_background="yes"
command_args="-nodaemon ${NETDATA_EXTRA_ARGS}"
start_stop_daemon_args="-u ${NETDATA_OWNER}"
```

then remove from depend:
```
checkpath -o ${NETDATA_OWNER} -d @localstatedir_POST@/cache/netdata /run/netdata
```

Add:
```
start_pre() {
    checkpath -o "${NETDATA_OWNER}" -d "@localstatedir_POST@/cache/netdata"
}
```

besides, I do not like the new script that downloads the configuration and waiting for termination, this is not normal behavior and should be removed in favor of a custom command, the getconf is a good example, the start_post and stop_post should be removed.